tutorial/normalizing_flows_for_atomic_solids_tutorial.ipynb

```diff
@@ -0,0 +1,582 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bkzoVxJtnO4k"
+      },
+      "source": [
+        "# Tutorial\n",
+        "\n",
+        "The goal of this tutorial colab is to train the normalizing flow model proposed in reference [1] on a small system comprising 8 particles of monatomic water in the cubic ice phase. \n",
```

Comment:
Since this is a tutorial it can be a good idea to be careful introducing abbreviations and the like. Here is the first mention of monatomic water and later it appears as mW, so I'd put here "monatomic water (mW)"

Reply:
Done.